code/modules/surgery/operations/operation_bone_repair.dm

```diff
@@ -27,9 +28,9 @@
 	display_results(
 		surgeon,
 		limb.owner,
-		span_notice("You begin to repair the fracture in [limb.owner]'s [limb.plaintext_zone]..."),
-		span_notice("[surgeon] begins to repair the fracture in [limb.owner]'s [limb.plaintext_zone] with [tool]."),
-		span_notice("[surgeon] begins to repair the fracture in [limb.owner]'s [limb.plaintext_zone]."),
+		span_notice("You begin to repair the fracture in [limb.owner ? "[limb.owner]'s [limb.plaintext_zone]" : limb]..."),
+		span_notice("[surgeon] begins to repair the fracture in [limb.owner ? "[limb.owner]'s [limb.plaintext_zone]" : limb] with [tool]."),
+		span_notice("[surgeon] begins to repair the fracture in [limb.owner ? "[limb.owner]'s [limb.plaintext_zone]" : limb]."),
```

Comment:
I think this deserves a macro with how often it's now copied across all files
`#define FORMAT_LIMB_OWNER(limb) (limb.owner ? "[limb.owner]'s [limb.plaintext_zone]" : limb)`